lib/routes/authorization.js

```diff
@@ -76,12 +77,13 @@ function generateCode(claims, client, scope, req) {
   });
 }
 
-function generateGrant(claims, client, scope) {
+function generateGrant(claims, client, scope, request) {
```

Comment:
please name this param `req` for consistency with the `generateCode` function above.  Or rename the one in `generateCode` to match, if you feel strongly about it :-)